javascript/javascript3/week3/Promises.js

```diff
@@ -0,0 +1,40 @@
+const arrUsers = ['santhoshboinapally', 'sunilkumarmahajan', 'suvarna-ratna']
+function renderRepo(user){
+    return fetch(`https://api.github.com/search/repositories?q=user:${user}`).then(res => res.json())
+        .then(data => {
+            const body = document.querySelector('body')
```

Comment:
try not to mix front-end code (manipulation of the DOM) with data processing. For example there could be `fetchRepo(user)` function called from here, which returns a promise to a result JSON.

Reply:
Let me know if that's not clear. But overall the code looks correct.